Build failed with 'Received status code 501 from server: HTTPS Required' message (Maven Central repo requires HTTPS now)

Today kafka-backup build fails with following errors.

After quick googling I found Maven Central repo requires HTTPS now:
https://support.sonatype.com/hc/en-us/articles/360041287334

> Effective January 15, 2020, The Central Repository no longer supports insecure communication over plain HTTP and requires that all requests to the repository are encrypted over HTTPS.

Could you please update build settings accordingly?

Build error output:
```
+ git clone https://github.com/itadventurer/kafka-backup.git
Cloning into 'kafka-backup'...
+ cd kafka-backup
+ ./gradlew shadowJar
Downloading https://services.gradle.org/distributions/gradle-5.1.1-all.zip
............................................................................................................................
Unzipping /root/.gradle/wrapper/dists/gradle-5.1.1-all/97z1ksx6lirer3kbvdnh7jtjg/gradle-5.1.1-all.zip to /root/.gradle/wrapper/dists/gradle-5.1.1-all/97z1ksx6lirer3kbvdnh7jtjg
Set executable permissions for: /root/.gradle/wrapper/dists/gradle-5.1.1-all/97z1ksx6lirer3kbvdnh7jtjg/gradle-5.1.1/bin/gradle

Welcome to Gradle 5.1.1!

Here are the highlights of this release:
 - Control which dependencies can be retrieved from which repositories
 - Production-ready configuration avoidance APIs

For more details see https://docs.gradle.org/5.1.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':shadowJar'.
> Could not resolve all dependencies for configuration ':runtimeClasspath'.
   > Could not resolve org.slf4j:slf4j-api:1.7.26.
     Required by:
         project :
      > Could not resolve org.slf4j:slf4j-api:1.7.26.
         > Could not get resource 'http://repo.maven.apache.org/maven2/org/slf4j/slf4j-api/1.7.26/slf4j-api-1.7.26.pom'.
            > Could not GET 'http://repo.maven.apache.org/maven2/org/slf4j/slf4j-api/1.7.26/slf4j-api-1.7.26.pom'. Received status code 501 from server: HTTPS Required
   > Could not resolve com.fasterxml.jackson.core:jackson-core:2.9.9.
     Required by:
         project :
      > Skipped due to earlier error
   > Could not resolve com.fasterxml.jackson.core:jackson-databind:2.9.9.
     Required by:
         project :
      > Could not resolve com.fasterxml.jackson.core:jackson-databind:2.9.9.
         > Could not get resource 'http://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.9.9/jackson-databind-2.9.9.pom'.
            > Could not GET 'http://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.9.9/jackson-databind-2.9.9.pom'. Received status code 501 from server: HTTPS Required
   > Could not resolve net.sf.jopt-simple:jopt-simple:6.0-alpha-3.
     Required by:
         project :
      > Skipped due to earlier error

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 39s
```

--- a/build.gradle
+++ b/build.gradle
@@ -18,7 +18,7 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 repositories {
-	maven { url "http://repo.maven.apache.org/maven2" }
+	maven { url "https://repo.maven.apache.org/maven2" }
 }
 
 dependencies {

